src/jit/instr.cpp

```diff
@@ -1100,12 +1100,6 @@ void CodeGen::inst_RV_TT_IV(instruction ins, emitAttr attr, regNumber reg1, GenT
                 {
                     varNum = addr->AsLclVarCommon()->GetLclNum();
                     offset = 0;
-
-                    // Ensure that all the GenTreeIndir values are set to their defaults.
```

Comment:
CC. @CarolEidt, had to remove this from `CodeGen::inst_RV_TT_IV`. It had been removed from the other paths since `memIndir` is `nullptr` when `rmOp` is not an indir node (such as if it is a HWIntrinsic memory load instead).

Reply:
Makes sense. Thanks.